src/test/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpointUnitTest.java

```diff
@@ -312,6 +313,35 @@ public void shouldRejectFulfilBySmsWithIncorrectRequestBody() throws Exception {
     verifyRejectedSmsFulfilmentRequest(json, url);
   }
 
+  /** Test should send out a new case event (register) */
```

Comment:
In order to test all the validation rules using `javax.validation.constraints` annotations in the `NewCaseDTO` we need quite a few more unit tests here, eg `firstName` cannot be blank etc etc